devops/docker/tasking-manager/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN apt-get update \
 # Upgrade pip
 RUN pip install --upgrade pip
 
-ARG branch=develop
+ARG branch=feature/1678-translations
```

Comment:
This needs to be changed back to `develp` before merging in. But should stay as it is for testing purposes.